lms/templates/courseware/progress.html

```diff
@@ -50,29 +50,45 @@
-                %endif
-             %else:
-                <button disabled="disabled">${_('Create Your Certificate')}</button>
-                <p class="text-center">${_("Your certificate will be available when you pass the course.")}</p>
-             %endif
-           </div>
-           <div id="errors-info" class="text-center"></div>
-       %endif
+            %if show_generate_cert_btn:
+                <div class="wrapper-msg wrapper-auto-cert">
+                    <div class="auto-cert-message" id="course-success">
+                        <div class="has-actions">
+                        %if passed:
+                            % if is_downloadable and download_url:
+                            <div class="msg-content">
+                                <h2 class="title">Your certificate is available</h2>
+                                <p class="copy">You can now download your certificate as a PDF. If you keep working and receive a higher grade, you can <a href="#">request an updated certificate</a>.</p>
+                            </div>
+                            <div class="nav-actions">
+                                <a class="btn" href="${download_url}" target="_blank" title="${_('PDF will open in a new browser window or tab.')}">
+                                    ${_("Download Your Certificate")}
+                                </a>
+                            </div>
+                            %elif is_generating:
+                            <div class="msg-content">
+                                <h2 class="title">We're working on it...</h2>
+                                <p class="copy">We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard whe it is ready.</p>
+                            </div>
+                            <div class="nav-actions">
+                                <button class="btn" disabled="disabled">${_('Create Your Certificate')}</button>
+                            </div>
+                            %else:
+                            <div class="msg-content">
+                                <h2 class="title">Congratulations, you qualified for a certificate!</h2>
+                                <p class="copy">You can keep working for a higher grade, or request your certificate now.</p>
+                            </div>
+                            <div class="nav-actions">
+                                <button class="btn" data-endpoint="${reverse('generate_user_cert', args=[unicode(course.id)])}" id="btn_generate_cert">${_('Request Certificate')}</button>
+                            </div>
+                            %endif
+                        %endif
+                    </div>
+                    <div id="errors-info" class="text-center"></div>
```

Comment:
@wedaly Can you shed some light on what might go in this errors-info div? Is this something related to the certs generation? And if so, what kind of content will be put in there, paragraph(s), lists, headers...? We will need to style it.